README.md

```diff
@@ -1 +1,128 @@
-# TODOLIST
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>Welcome...</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [TO-DO-LIST] <a name="about-project"></a>
+
+My to-do list project in Microverse.
+
+**[TO-DO-LIST]** is a project experience that Microverse offers to help nuture your skills in designing on different screen sizes.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ 1. HTML5
+ 2. CSS3
+ 3. JavaScript
+
+
+### Key Features <a name="key-features"></a>
+
+  1. It is made to cater for mobile devices and desktops
+  2. Introductory page
+  3. Preview of the list page
+  4. Responsive design
+
+### Live Demo <a name="live-demo"></a>
+
+[Live Demo](not available)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+An extension to your browser
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone [TO-DO-LIST](https://github.com/2004-okari/TODOLIST)
+### Install
+
+Install this project with:
+
+cd MY-PORTFOLIO
+npm install
+### Usage
+
+To run the project, execute the following command:
+
+Open the index.html file on the browser or use a VS code extension server to start the project`
+
```

Comment:
- [ ] This part of the readme doesnt fully provide a guide on how to run the project once the user has cloned it please kindly provide adequate steps and information on how to do that. see below the provided example 👍 
- example
- > After installing the project, run the following command:
- run `npm i` or `npm install` in your terminal to download all the packages needed to make the project run
- run `npm run start` or `npm start` to open the project in your browser 
- * This is all that should be required of you to assist anyone using your app for the first time whether it is a developer or just a random user 👍